.github/workflows/sycl_windows_build_and_test.yml

```diff
@@ -50,9 +70,23 @@ jobs:
           --cmake-opt="-DCMAKE_C_COMPILER_LAUNCHER=sccache"
     - name: Build
       id: build
+      shell: cmd
+      # TODO replace utility installation with a single CMake target
       run: |
-        cmake --build build --target sycl-toolchain
-        sccache --show-stats
+        cmake --build build --target deploy-sycl-toolchain
+        cmake --build build --target utils/FileCheck/install
+        cmake --build build --target utils/count/install
+        cmake --build build --target utils/not/install
+        cmake --build build --target utils/lit/install
+        cmake --build build --target utils/llvm-lit/install
+        cmake --build build --target install-clang-format
+        cmake --build build --target install-clang-tidy
```

Comment:
> I see that I missed the same change in #4911.

Just FYI. I just realized that we need to install clang-format on Linux because `lint` check uses clang-format from our Linux nightly build. There is no use for clang-format on Windows though.